### ISSUE TITLE
Breaking 'Search Everywhere' dialog window for projects without git

### ISSUE THREAD

**Issue by elivin** [OpherV/gitflow4idea#265]
* **I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] puppy => You're not submitting a puppy. I already have one and he's [adorable](https://twitter.com/Opherv/status/974341249099542528)

* **What is the current behavior?**
I have some projects that used mercurial as VCS.
When I search by 'Search Everywhere' or 'Find Action' functions, sometimes I have NPE for gitflow4idea plugin. It makes 'Init repo' action.

* **Is this a bug? Sorry about that. If so give me explicit details how to reproduce:**

1. Open IntelliJ
2. Open project that not used git
3. Double Shift (of Ctrl+Shift+A)
4. Type _ini_ or _rep_
5. See to IDE error notification
![gitflow-init-action](https://user-images.githubusercontent.com/12787111/70844124-4022c100-1e5e-11ea-9ced-08d5e5e19e7d.gif)

* **What is the expected behavior?**
Do nothing on update actions if git repository is not initialized.

* **What is the motivation / use case for changing the behavior?**

* **Please tell me about your environment:**
  
  - Gitflow4idea version: *0.7.1*
  - Gitflow version: *null* :)
  
  - *IntelliJ Help -> about > click copy icon and paste here. Should look like this:*
  ~~~~
    PyCharm 2019.3 (Professional Edition)
    Build #PY-193.5233.109, built on November 28, 2019
    Runtime version: 11.0.4+10-b520.11 amd64
    VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
    Linux 4.19.0-6-amd64
    GC: ParNew, ConcurrentMarkSweep
    Memory: 2014M
    Cores: 4
    Registry: debugger.watches.in.variables=false
    Non-Bundled Plugins: Gitflow, com.alayouni.ansiHighlight, com.intellij.ideolog, de.netnexus.camelcaseplugin, izhangzhihao.rainbow.brackets, name.kropp.intellij.makefile, net.ashald.envfile, org.toml.lang, ru.adelf.idea.dotenv
  ~~~~

* **Other information** (e.g. detailed explanation, stacktrace, related issues, suggestions how to fix, links for me to have context words of praises, pictures of puppies (again with the puppy??) )

Stacktrace:
```
java.lang.NullPointerException
	at gitflow.actions.ReInitRepoAction.update(ReInitRepoAction.java:23)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:177)
	at com.intellij.ide.util.gotoByName.GotoActionModel.updateActionBeforeShow(GotoActionModel.java:303)
	at com.intellij.ide.util.gotoByName.GotoActionModel$ActionWrapper.lambda$getPresentation$0(GotoActionModel.java:652)
	at com.intellij.ide.util.gotoByName.GotoActionModel.lambda$updateOnEdt$4(GotoActionModel.java:456)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:309)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:441)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:424)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:407)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:906)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:779)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:422)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:698)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:421)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

PR https://github.com/OpherV/gitflow4idea/pull/264 is fix for this, not #249 :)

**Comment by OpherV** [comment]
Good catch! I'll add this to my test cases

**Comment by OpherV** [comment]
Fixed in 0.7.2
